### PR TITLE
add high level loc design spec

### PR DESCRIPTION
**Pull request by OsmondJiang**

### CODE REVIEW

**Comment by OsmondJiang** [comment]
#3343

File: docs/designs/loc.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Localization Support
+
+## Description
+
+DocFX build supports localization contents, there are a few features need to be supported for localization contents:
+  - Partial contents: the localization contents are often a part of source content.
+  - Localization contents are not required to be stored in the same repo of source content, they can be everywhere.
```
Comment (live1206) [inline docs/designs/loc.md]: >everywhere [](start = 104, length = 10)

anywhere?

File: docs/designs/loc.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - Localization publishing may have a few specific requirements which are different with source publishing, they need overwrite the source configuration
```
Comment (live1206) [inline docs/designs/loc.md]: >need overwrite [](start = 114, length = 14)

need to overwrite

File: docs/designs/loc.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - Focus the Loc publishing configuration controlling, like bilingual or contribution.  
+  - Source configuration changing will be immediately applied to Loc, no more manually sync.
+
+### [URL Schema and File Output](https://github.com/dotnet/docfx/blob/v3/docs/designs/output.md#url-schema)
+
+Follow the spec defined at [here](https://github.com/dotnet/docfx/blob/v3/docs/designs/output.md#url-schema).
+
+### Mappings between source content and loc content
+
+Below kinds of mappings are considered to be supported:
```
Comment (live1206) [inline docs/designs/loc.md]: >Below kinds of mappings [](start = 0, length = 23)

The mappings below

**Pull request merged**